python/tvm/tir/schedule/schedule.py

```diff
@@ -2479,6 +2480,31 @@ def transform_layout(
             primitive will be called in addition to the
             TransformLayout primitive.
 
+        pad_value: Optional[Union[int, float, PrimExpr, IndexMap, Callable]]
```

Comment:
Document the assumption when pad_value is IndexMap. I remember in the RFC we assume it should contain no BufferLoad from buffers except the current buffer

Reply:
Thank you, and the docstring has been updated.  I've also added two unit tests, one that validates that an error is raised when the pad value loads from a different buffer, and one that specifies the intended behavior for pad value that loads from the transformed buffer.  The latter is currently marked with `pytest.mark.xfail`, as the support isn't implemented yet.